WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Price/Product+PriceSettingsViewModels.swift

```diff
@@ -0,0 +1,35 @@
+import Yosemite
+
+extension Product {
+    func createRegularPriceViewModel(using currencySettings: CurrencySettings,
```

Comment:
I refactored in my other PRs from creating a view model with `Product`'s property to a static function with the property passed in as an argument (like `static func createRegularPriceViewModel(price: String, using currencySettings: CurrencySettings, ...)`), since the `Product`'s properties could be changed on the settings screen and if we create a view model again (like when the cell is reused from scrolling) the value could be outdated